exo-go/src/process_helpers/index.go

```diff
@@ -38,6 +40,17 @@ func Start(dir string, commandWords ...string) (*exec.Cmd, io.WriteCloser, *byte
 	return cmd, in, &out, nil
 }
 
+// Wait waits for the given text for the specified duration
+func Wait(stdout fmt.Stringer, text string, done func()) {
+	ticker := time.NewTicker(100 * time.Millisecond)
+	var output string
+	for !strings.Contains(output, text) {
```

Comment:
Here might be a reason why the Go version is slower. The Node version evaluates the text right when it is written into the stream, and calls the callback when it finds the text right away. Your method here waits 100ms, then checks, then waits another 100ms. These delays can add up.